common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -175,6 +175,13 @@ public class RssBaseConf extends RssConf {
           .defaultValue(false)
           .withDescription("Whether enable visiting secured hadoop cluster.");
 
+  public static final ConfigOption<Boolean> RSS_SECURITY_HADOOP_KERBEROS_PROXY_USER_ENABLE =
+      ConfigOptions.key("rss.security.hadoop.kerberos.proxy.user.enable")
+          .booleanType()
+          .defaultValue(true)
+          .withDescription(
+              "Whether create proxy user for spark job user to visiting secured hadoop cluster.");
```

Comment:
description: `Whether using proxy user for spark job user to access secured Hadoop cluster`

Reply:
done